### PR TITLE
feat: move recovery phrase into 'settings'

### PR DESCRIPTION
**Pull request by kathaypacific**
Reverts valora-inc/wallet#2808

### Description

This PR:
- fixes the issue of incorrect drawer top bar shown on the recovery phrase screen which led to a crash on press
- rename the navigation variable to be more explicit
- after completing the back up quiz, navigate home instead of back to recovery phrase screen (discussed with Ellen)

### Other changes

N/A

### Tested

Manually

### How others should test

See details in https://github.com/valora-inc/wallet/pull/2758

In alfajores, for a new user who hasn't backed up their recovery phrase:
- they should see the recovery phrase items in the drawer menu AND the settings screen
For a user who has backed up their recovery phrase:
- they should only see the recovery phrase item in the settings screen
The recovery phrase functionality should remain unchanged (only the entry points to where you can start this flow has changed here)

### Related issues

- Fixes RET-302

### Backwards compatibility

Yes

### CODE REVIEW

File: src/backup/BackupComplete.tsx [hunk 1]
```diff
@@ -32,7 +32,7 @@ function BackupComplete({ route }: Props) {
         navigate(Screens.Settings, { promptConfirmRemovalModal: true })
       } else if (backupCompleted) {
         ValoraAnalytics.track(OnboardingEvents.backup_complete)
-        navigate(Screens.BackupIntroduction)
+        navigateHome()
```
Comment (kathaypacific) [inline src/backup/BackupComplete.tsx:35]: this change was discussed with Ellen

File: src/navigator/Navigator.tsx [hunk 2]
```diff
@@ -404,6 +405,11 @@ const backupScreens = (Navigator: typeof Stack) => (
       component={AccounSetupFailureScreen}
       options={AccounSetupFailureScreen.navOptions}
     />
+    <Navigator.Screen
+      name={Screens.BackupIntroduction}
```
Comment (kathaypacific) [inline src/navigator/Navigator.tsx:409]: we now have 2 screens by the same name - should i separate them? one is in the drawer navigator and this is in the stack, the navigation seems to work fine like this but not sure if it's strictly the best thing to do 😅

File: test/schemas.ts [hunk 3]
```diff
@@ -1542,7 +1542,7 @@ export const v70Schema = {
   },
   app: {
     ...v69Schema.app,
-    createAccountCopyTestType: 'ACCOUNT',
+    shouldShowRecoveryPhraseInSettings: false,
```
Comment (jeanregisser) [inline test/schemas.ts:1545]: I'm confused, normally we shouldn't change previous schemas.

File: test/schemas.ts [hunk 4]
```diff
@@ -1557,9 +1557,24 @@ export const v71Schema = {
...
+  ...v71Schema,
+  _persist: {
+    ...v71Schema._persist,
+    version: 72,
+  },
+  app: {
+    ...v71Schema.app,
+    maxSwapSlippagePercentage: 2,
+    swapFeeEnabled: false,
+    swapFeePercentage: 0.743,
```
Comment (jeanregisser) [inline test/schemas.ts:1574]: Maybe a bad merge?

File: src/navigator/Navigator.tsx (hunk 2 above)
Comment (jeanregisser) [reply src/navigator/Navigator.tsx:409]: 😄 should be ok if navigation works as expected.

**Review by jeanregisser — APPROVED** [review]
Looking great! 🚀

File: test/schemas.ts (hunk 4 above)
Comment (kathaypacific) [reply test/schemas.ts:1574]: oh my goodness the merge was so bad! i will untangle this mess...

Comment (kathaypacific) [reply test/schemas.ts]: thanks for picking up the bad merge @jeanregisser ! i have untangled the mess :D

**Comment by ValoraQA** [comment]
Hey @MuckT we have verified the above task on Latest iOS Alfajores Build V 1.40.0 and updated status in Test Rails

**Observation :** 
- When user has not set up the recovery phrase, it is seen in Hamburger Menu
- When user has set up the recovery Phrase , it is seen in settings under security option"
- Extra space is observed on recovery phrase once user has completed the setup and also "<" is seen instead of hamburger menu and we have raised bug for it

**New bug :** [link](https://github.com/valora-inc/wallet/issues/2844)

**Test Rail link:**
**iOS 14:** https://valoraapp.testrail.io/index.php?/tests/view/69518

**Device:** iPhone 12(14.7.1)

Cc: @kathaypacific 

Thanks.!

**Comment by MuckT** [comment]
@ValoraQA [Extra space is seen on "Your Recovery Phrase" page after user has completed setting up of the recovery phrase #2844](https://github.com/valora-inc/wallet/issues/2844) - Looks like a valid bug. @kathaypacific did you want to take a look? It's currently a P4.

**Comment by ValoraQA** [comment]
Hey @MuckT we have verified the above task on Latest Android Alfajores Build V 1.40.0 and updated status in Test Rails

**Observation** :


- When user has not set up the recovery phrase, it is seen in Hamburger Menu
- When user has set up the recovery Phrase , it is seen in settings under security option"
- Extra space is observed on recovery phrase once user has completed the setup and also "<" is seen instead of hamburger menu and we have raised bug for it

**New bug :** [link](https://github.com/valora-inc/wallet/issues/2844)

**Test Rail link:**
**Android 12**: https://valoraapp.testrail.io/index.php?/tests/view/69580

**Device**: Google Pixel 4a(12.0)

**CC**: @kathaypacific

Thanks.!

**Pull request merged**